docs/src/content/en/docs/workflows/overview.mdx

```diff
@@ -1,18 +1,30 @@
 ---
 title: "Handling Complex LLM Operations | Workflows | Mastra"
-description: "Workflows in Mastra help you orchestrate complex sequences of operations with features like branching, parallel execution, resource suspension, and more."
+description: "Workflows in Mastra help you orchestrate complex sequences of tasks with features like branching, parallel execution, resource suspension, and more."
 ---
 
 import { Steps } from "nextra/components";
 
 # Workflows overview
 
-Workflows let you define and orchestrate complex sequences of tasks as **typed steps** connected by data flows. Each step has clearly defined inputs and outputs validated by Zod schemas.
-
-A workflow manages execution order, dependencies, branching, parallelism, and error handling — enabling you to build robust, reusable processes. Steps can be nested or cloned to compose larger workflows.
+Workflows let you define and orchestrate complex sequences of tasks as **typed steps** connected by data flows. While a single agent may be suitable for simple tasks, real-world applications often involve multiple steps with different logic, tools, external services, or context requirements.
 
 ![Workflows overview](/image/workflows/workflows-overview.jpg)
 
+## Real-world example
```

Comment:
I think "when to use a workflow" would be a more helpful h2

As an aside -  in the future, we need a guide on agents vs. workflows, then we can link to it and keep this page more terse